frontend/src/app/modules/common/editable-toolbar-title/editable-toolbar-title.sass

```diff
@@ -19,3 +19,19 @@
 
   &.-changed
     font-style: italic
+
+#inputTextID 
```

Comment:
I am strong for keeping the styles the way they were before. Unfortunately this was not clearly described in the ticket, it's on our head. I'm sorry about that. So I'm afraid we don't need any of the styles from this file.